### PR TITLE
Fix release job centos6

### PR DESCRIPTION
**Pull request by tylarb**
Missed a hard coded check for centos 6 in release job. This should resolve failures in release job caused by this check.

### CODE REVIEW

File: concourse/scripts/verify_gpdb_versions.bash [hunk 1]
```diff
@@ -25,7 +25,7 @@ yum -d0 -y install git
 GREENPLUM_INSTALL_DIR=/usr/local/greenplum-db-devel
 GPDB_SRC_SHA=$(cd gpdb_src && git rev-parse HEAD)
 
-for bin_gpdb in bin_gpdb_{centos{6,7},ubuntu18.04}; do
+for bin_gpdb in bin_gpdb_{centos7,ubuntu18.04}; do
```
Comment (bradfordboyle) [inline concourse/scripts/verify_gpdb_versions.bash:28]: Do we want to hard code the list of platforms here? Or would it make sense to do `for bin_gpdb in bin_gpdb_*`?

**Comment by d** [comment]
I'm starting to review this

**Comment by d** [comment]
:clap: for updating the README!

> When centos 6 was removed from the pipeline <snip>
nitpick: It'd be best to explicitly call out commit 39c95cb7496cee (or PR #9981) in the commit that mentions the oversight

File: concourse/scripts/verify_gpdb_versions.bash (hunk 1 above)
Comment (d) [reply concourse/scripts/verify_gpdb_versions.bash:28]: So the pros of using a glob here is not needing a change. What are the cons? @bradfordboyle

Comment (tylarb) [reply concourse/scripts/verify_gpdb_versions.bash:28]: @d cons are greedy bash-isms. Perhaps this check is actually different in the future for different platforms.

Unsure if specifically a con or not, but we already hard code platforms generously throughout the rest of the CI.

Comment (tylarb) [reply concourse/scripts/verify_gpdb_versions.bash:28]: @d do you have a hard preference to use a glob? if not, we can go ahead and merge.

**Comment by d** [comment]
LGTM

**Pull request merged**